### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by CongMa13**
- Updated target list
- Add emulation test docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,12 +21,12 @@ The test suite includes validation and benchmarking projects that focus on unit
...
 
-* CDNA class GPU featuring matrix core support: gfx908, gfx90a, gfx940, gfx940, gfx942 as 'gfx9'
-* RDNA3 class GPU featuring AI acceleration support: gfx1100, gfx1101, gfx1102 as 'gfx11'
+* CDNA class GPU featuring matrix core support: gfx908, gfx90a, gfx940, gfx941, gfx942 as 'gfx9'
+* RDNA3 class GPU featuring AI acceleration support: gfx1100, gfx1101, gfx1102 as 'gfx11' and gfx1200, gfx1201 as 'gfx12'
 
 Dependencies:
 
-* Minimum ROCm version support is 6.3.
+* Minimum ROCm version support is 6.4.
```
Comment (CongMa13) [inline README.md:29]: should change it to 6.4?

File: docs/api-reference/api-reference-guide.rst [hunk 2]
```diff
@@ -268,7 +271,6 @@ layout_t
 ^^^^^^^^
 
 .. doxygenenum:: rocwmma::layout_t
-   :members:
```
Comment (CongMa13) [inline docs/api-reference/api-reference-guide.rst:271]: doxygenenum does not have members property. The output is correct after removing it.

image

File: docs/api-reference/api-reference-guide.rst [hunk 3]
```diff
@@ -315,3 +317,14 @@ Sample programs
 
 See a sample code for calling rocWMMA functions ``load_matrix_sync``, ``store_matrix_sync``, ``fill_fragment``, and ``mma_sync`` `here <https://github.com/ROCm/rocWMMA/blob/develop/samples/simple_hgemm.cpp>`_.
 For more such sample programs, refer to the `Samples directory <https://github.com/ROCm/rocWMMA/tree/develop/samples>`_.
+
```
Comment (CongMa13) [inline docs/api-reference/api-reference-guide.rst:317]: image

File: docs/install/installation.rst [hunk 4]
```diff
@@ -214,6 +218,8 @@ Below are the project options available to build rocWMMA library with or without
         -   Use system Google Test library instead of downloading and building it
         -   OFF (requires ROCWMMA_BUILD_TESTS=ON)
 
+.. [1] The default AMDGPU_TARGET is: ``ggfx908;gfx90a;gfx942;gfx1100;gfx1101;gfx1102;gfx1200;gfx1201``
```
Comment (CongMa13) [inline docs/install/installation.rst:221]: image

File: docs/api-reference/api-reference-guide.rst [hunk 5]
```diff
@@ -315,3 +314,14 @@ Sample programs
...
+Emulation tests
+---------------
+
+The emulation test is a smaller test suite specifically designed for emulators. It comprises a selection of test cases from the full ROCWMM test set, allowing for significantly faster execution on emulated platforms. Despite its concise nature, the emulation test supports ``smoke``, ``regression``, and ``extended`` modes.
```
Comment (amd-jnovotny) [inline docs/api-reference/api-reference-guide.rst:321]: Just want to confirm this should be ROCWMM and not ROCWMMA?

File: docs/api-reference/api-reference-guide.rst [hunk 6]
```diff
@@ -315,3 +314,14 @@ Sample programs
...
+
+For example, run a smoke test.
```
Comment (amd-jnovotny) [inline docs/api-reference/api-reference-guide.rst:323]:
```suggestion
For example, the following command runs the smoke test.
```

**Review by amd-jnovotny — APPROVED** [review]
Docs changes look good overall: one question and one editorial suggestion.

File: docs/api-reference/api-reference-guide.rst (hunk 5 above)
Comment (cgmillette) [reply docs/api-reference/api-reference-guide.rst:321]: Should be rocWMMA

Comment (cgmillette) [inline docs/api-reference/api-reference-guide.rst:321]:
```suggestion
The emulation test is a smaller test suite specifically designed for emulators. It comprises a selection of test cases from the full rocWMMA test set, allowing for significantly faster execution on emulated platforms. Despite its concise nature, the emulation test supports ``smoke``, ``regression``, and ``extended`` modes.
```

**Comment by CongMa13** [comment]
duplicated with #471

**Pull request closed without merging**